web-frontend/src/store/index.js

```diff
@@ -98,7 +100,7 @@ export default new Vuex.Store({
       state.repo = analysis.repository;
       state.commit = analysis.commit;
       state.date = new Date(analysis.timestamp).toString();
-      state.change_summary = analysis.rust_dependencies.change_summary || {};
+      state.change_summary = analysis.rust_dependencies.change_summary || {new_updates:[], new_rustsec:{vulnerabilities:[], warnings:[]}};
```

Comment:
good job, I think that was the issue : )

Reply:
lgtm